client/lib/analytics/index.js

```diff
@@ -151,7 +159,9 @@ const analytics = {
 			if ( process.env.NODE_ENV !== 'production' ) {
 				for ( const key in eventProperties ) {
 					if ( isObjectLike( eventProperties[ key ] ) && typeof console !== 'undefined' ) {
-						console.error( `Unable to record event "${ eventName }" because nested properties are not supported by Tracks. Check '${ key }' on`, eventProperties ); //eslint-disable-line no-console
+						const errorMessage = `Unable to record event "${ eventName }" because nested 
+							properties are not supported by Tracks. Check '${ key }' on`;
```

Comment:
might be useful to split this string to make it more legible

```js
const errorMessage = (
	`Unable to record event "${ eventName }" because nested ` +
	`properties are not supported by Tracks. Check '${ key }' on `
);
```

also check:
 - we're using double-quotes for `eventName` but single-quotes for `key`
 - the string mysteriously ends

maybe we could log `eventProperties` in a more defined way or in a separate log statement